internal/driver/glfw/loop_desktop.go

```diff
@@ -0,0 +1,90 @@
+// +build !js,!wasm,!web
```

Comment:
Possibly here as well the duplication could be less. The core loop is valid, the specific files perhaps could be just helper files (with functions such as init, poll and stop - the rest is identical I think.